src/frontend/sharedComponents/ProjectInviteBottomSheet/NewInviteBottomSheetContent.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react';
+import {BottomSheetContent} from '../BottomSheetModal';
+import {View} from 'react-native';
+import {LIGHT_GREY} from '../../lib/styles';
+import InviteIcon from '../../images/AddPersonCircle.svg';
+import {defineMessages, useIntl} from 'react-intl';
+
+const m = defineMessages({
+  declineInvite: {
+    id: 'sharedComponents.ProjectInviteBottomSheet.NewInviteBottomSheetContent.declineInvite',
+    defaultMessage: 'Decline Invite',
+  },
+  acceptInvite: {
+    id: 'sharedComponents.ProjectInviteBottomSheet.NewInviteBottomSheetContent.acceptInvite',
+    defaultMessage: 'Accept Invite',
+  },
+  joinProject: {
+    id: 'sharedComponents.ProjectInviteBottomSheet.NewInviteBottomSheetContent.joinProject',
+    defaultMessage: 'Join Project {projName}',
+  },
+  invitedToJoin: {
+    id: 'sharedComponents.ProjectInviteBottomSheet.NewInviteBottomSheetContent.invitedToJoin',
+    defaultMessage: "You've been invited to join {projName}",
+  },
+});
+
+type NewInviteBottomSheetContentProp = {
+  isLoading?: boolean;
+  handleReject: () => void;
+  handleAccept: () => void;
+  projectName?: string;
+};
+
+export const NewInviteBottomSheetContent = ({
```

Comment:
nit: extract styles to stylesheet

Reply:
https://github.com/digidem/comapeo-mobile/pull/366/commits/a5e25113e5b74d9b925c9b6f8fc79639206a0e4e